src/components/exercise/part-card.cjsx

```diff
@@ -141,7 +141,11 @@ ExerciseStepCard = React.createClass
           exercise_uid={step.content?.uid}
           related_content={related_content}/>
 
-    cardClasses = classnames 'task-step', 'openstax-exercise-card', className
+    cardClasses = classnames 'task-step', 'openstax-exercise-card', className, {
+      "deleted-homework": @props.task.type is 'homework' and @props.task.is_deleted
+      "deleted-reading": @props.task.type is 'reading' and @props.task.is_deleted
+    }
```

Comment:
same as in [`exercise/index.cjsx`](https://github.com/openstax/react-components/pull/101/files#r72507694), should these be `?`-y?